WooCommerce/UITestsFoundation/XCTest+Extensions.swift

```diff
@@ -114,8 +114,44 @@ extension XCUIElement {
         startCoordinate.press(forDuration: 0.01, thenDragTo: destination)
     }
 
-    public func getTextVisibilityCount(text: String) throws -> Int {
-        let predicate = NSPredicate(format: "label CONTAINS[c] %@", text)
+    func getStaticTextVisibilityCount(textToFind: String) throws -> Int {
+        let predicate = NSPredicate(format: "label CONTAINS[c] %@", textToFind)
         return XCUIApplication().staticTexts.containing(predicate).count
     }
+
+    public func assertTextVisibilityCount(textToFind: String) {
+        let count = try! getStaticTextVisibilityCount(textToFind: textToFind)
+        XCTAssertTrue(count == 1, "Expected '\(textToFind)' to appear once, but it appeared '\(count)' times!")
+    }
+
+    // Parent is using accessibilityIdentifier, child is using staticText
+    func verifyChildElementOnParentCell(parent: String, child: String) throws -> Bool {
+        let parentPredicate = NSPredicate(format: "identifier == %@", parent)
+        let childPredicate = NSPredicate(format: "label ==[c] %@", child)
+
+        return XCUIApplication().tables.cells.matching(parentPredicate).children(matching: .staticText).element(matching: childPredicate).firstMatch.exists
+    }
+
+    public func assertElementExistsOnCell(mainCell: String, elementToFind: String) {
+        XCTAssertTrue(try verifyChildElementOnParentCell(parent: mainCell, child: elementToFind), "Element does not exist on cell!")
+    }
+
+    func verifyLabelContainsTwoPredicates(firstSubstring: String, secondSubstring: String) throws -> Bool {
```

Comment:
Naming nitpick:

The fact that this method uses predicate seems like an implementation detail that the caller and reader don't need to know. What do you think of:

```suggestion
    func verifyLabelContains(substring firstSubstring: String, and secondSubstring: String) throws -> Bool {
```

This would be called like: `verifyLabelContains(substring: "foo", and: "bar")`. A more explicit alternative might be `verifyLabelContains(substring: "foo", andSubstring: "bar")`. Or, if we want to be more implicit instead: `verifyLabelContains("foo", and: "bar")`, which can be achieved replacing `substring` with `_` in the suggestion above.